core-tests/shared/src/test/scala/zio/stm/TArraySpec.scala

```diff
@@ -292,7 +292,7 @@ object TArraySpec extends ZIOBaseSpec {
         for {
           tArray    <- makeStair(N).commit
           findFiber <- tArray.findLast(_ % largePrime == 0).commit.fork
-          _         <- STM.foreach(0 until N)(i => tArray.update(i, _ => 1)).commit
+          _         <- STM.foreach(List.range(0, N))(i => tArray.update(i, _ => 1)).commit
```

Comment:
The old `0 until N` no longer works?

Reply:
Ranges work if it is an inclusive range but not an exclusive range. The same issue applies to `Future.traverse`. This seems like a bug. Opened an issue for it at https://github.com/scala/bug/issues/12105.